README.md

```diff
@@ -30,7 +30,7 @@ This repository contains different [Jupyter Notebooks](https://jupyter.org/) to
   * [Reading Online](#reading-online)
   * [Experimenting Locally](#experimenting-locally)
     * [Using Docker or Podman](#using-docker-or-podman)
-    * [On the Machine (Linux/x64)](#on-the-machine-linuxx64)
+    * [On the Machine (Linux/x64 & arm64)](#on-the-machine-linuxx64--arm64)
```

Comment:
I think you should state explicitly that this also runs on mac os (using dockers virtualization).

Reply:
At the Docker part?
I would expect Docker to directly mean: works where Docker works. 
No?

Or the Machine one? Does it directly work? Will it get the right executable by itself?

Reply:
Sorry. Was mixing the sections up. My bad!